### DISCUSSION TITLE
Disable flip when Combobox is open

### DISCUSSION THREAD

**Discussion by patrikholcak** [ariakit/ariakit#3780] [Questions]
Hey, I don’t know if this is a bug or not, but when I filter in combobox select, the combobox sometimes flips direction and I’m trying to (unsuccessfully) prevent it. Another thing is when the select closes and reopens it stays in the flipped position — which might be a bug. I want the select to automatically position itself but prevent it from flipping once it’s open and user starts typing to find an option.

I tried to conditionally disable `flip` or passing the preferred placements, but it doesn’t seem to work. I also tried calling `context.setState` with both `placement` and `currentPlacement` on both select and combobox contexts, but to no avail. I even tried delaying disabling the flip after all transitions are finished but that also doesn’t help

This is reproducible on the example page
https://ariakit.org/examples/select-combobox

With these window settings (just open devtools and resize)
```
window.innerHeight = 491
window.scrollY = 461
```

Here’s a video:

https://github.com/ariakit/ariakit/assets/72975/2633839c-165a-48be-afee-540179410aa6

As you can see, the select opens to the top but once I type and the options get filtered it gets positioned to the bottom and stays there when I reopen it

**Comment by diegohaz** [comment]
@atomiks Do you know if there's an option in Floating UI to disable `flip` specifically during element resize? I'm not sure if that would be the best solution, but it's the first thing that comes to mind.
